.github/workflows/update_file.yaml

```diff
@@ -0,0 +1,38 @@
+name: Update files
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '43 1 * * 1'
+
+jobs:
+
+  pull-from-sunbeam:
+    name: Pull files from sunbeam
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout local repository
+        uses: actions/checkout@v4
+
+      - name: set git config
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          git config --global user.email "${GITHUB_ACTOR_ID}+${GITHUB_ACTOR}@users.noreply.github.com"
+          git config --global user.name "$(gh api /users/${GITHUB_ACTOR} | jq .name -r)"
+          git config -l
+          
+      - name: Check pull from sunbeam
+        run: make sync-dashboards-alert-rules
```

Comment:
I do not see reason for using Makefile and shell script here, like this we are adding more places which potentially need to be changed. To we should use `actions/checkout@v4` to get sunbeam repo (we can use the `sparse-checkout` option to get only subdirectory) and simply copy files we need.

If I understand this workflow properly, we do not need to have any check if new file was created. If git diff is empty, no PR will be created. Is that right?